internal/store/leaf/recipients.go

```diff
@@ -111,16 +111,37 @@ func (s *Store) RemoveRecipient(ctx context.Context, id string) error {
 	nk := make([]string, 0, len(rs)-1)
 RECIPIENTS:
 	for _, k := range rs {
+
+		// First lets try a simple match of the stored ids
 		if k == id {
+			debug.Log("removing recipient based on id match %s", k)
 			continue RECIPIENTS
 		}
+
+		// If we don't match immediately, we may need to loop through the recipient keys to try and match.
+		// To do this though, we need to ensure that we also do a FindRecipients on the id name from the stored ids.
+		recipient_ids, err := s.crypto.FindRecipients(ctx, k)
+		if err != nil {
+			out.Printf(ctx, "Warning: Failed to get GPG Key Info for %s: %s", k, err)
```

Comment:
There is `out.Warnf`. Consider using that for consistent formating.

Reply:
Done.  I have updated an additional call that was already present and using `Printf` for consistency.